package.json

```diff
@@ -52,25 +52,25 @@
     "TypeScript"
   ],
   "dependencies": {
-    "eslint-config-standard": "^12.0.0"
+    "@typescript-eslint/parser": "^1.13.0",
```

Comment:
https://github.com/eslint/eslint/pull/11388/files#diff-33bbe98f532eb757e2789f1a3549c9caR41

The parser can and should be a `dependency`. Makes it easier for the user, right? #65.

A parser update seems necessary for ESLint 6. So I updated it to the latest version.